chapters/pre-recorded-stt/speaker-diarization.mdx

```diff
@@ -25,17 +25,20 @@ Breaking changes may still be introduced to this API, but an advanced notice wil
 We're looking for feedback to improve this feature, [share yours here](https://gladiaio.typeform.com/feedback?typeform-source=app.gladia.io).
 </Note>
 
-For improved diarization handling edge cases and challenging audio, you can enable enhanced diarization by using the `diarization_enhanced` parameter.
+For improved diarization handling edge cases and challenging audio, you can enable enhanced diarization by using the `enhanced` parameter in the `diarization_config` object.
 
 ```json
 {
   "audio_url": "<your audio URL>",
-  "diarization_enhanced": true
+  "diarization": true,
+  "diarization_config": {
+    "enhanced": true
+  },
 }
 ```
 
 ## Response
-When `diarization` or `diarization_enhanced` is enabled, each utterance will contain a `speaker` field, whose value is an index representing the speaker.
+When `diarization` or `enhanced diarization` is enabled, each utterance will contain a `speaker` field, whose value is an index representing the speaker.
```

Comment:
you should just simplify to "when `diarization` is enabled" because it works the same whether it is `enhanced` or not

Reply:
Correct!